### PR TITLE
Improve UX and style event source modal

### PR DESCRIPTION
**Pull request by jgoley**
screen shot 2016-12-12 at 4 06 06 pm

One #188 is merged I'll probably update the checkbox for 'suggest incident reports'

### CODE REVIEW

**Comment by nathanathan** [comment]
The save button doesn't work for me.

File: client/controllers/sourceModal.coffee [hunk 1]
```diff
@@ -147,51 +154,51 @@ Template.sourceModal.events
...
           daylightSavings = daylightSavings and moment.utc(
             date.year() + "-11-01") >= date
-          tz = if daylightSavings then "EDT" else "EST"
+          tz = if daylightSavings then 'EDT' else 'EST'
           date = date.utcOffset(UTCOffsets[tz])
-          templateInstance.$("#publishDateTZ").val(tz)
-          templateInstance.$('#publishDate').data("DateTimePicker").date(date)
-          templateInstance.$('#publishTime').data("DateTimePicker").date(date)
+          instance.$('#publishDateTZ').val(tz)
+          console.log instance.datePicker
```
Comment (nathanathan) [inline client/controllers/sourceModal.coffee]: Do you want to leave this in?

File: client/controllers/sourceModal.coffee [hunk 2]
```diff
@@ -147,51 +154,51 @@ Template.sourceModal.events
...
+  'click #suggested-articles li': (event, instance) ->
     event.preventDefault()
-    title = event.currentTarget.innerText
-    templateInstance.currentTitle.set(title)
-    url = event.currentTarget.getAttribute 'href'
-    input = templateInstance.find('#article')
-    input.value = url
-    titleInput = templateInstance.find('#title')
-    titleInput.value = title
+    console.log @
```
Comment (jgoley) [inline client/controllers/sourceModal.coffee]: My bad, will remove.

**Comment by nathanathan** [comment]
When editing reports the publication date gets charged to the current date.

**Comment by nathanathan** [comment]
👍 Looks good

**Pull request merged**